pkg/trace/api/info_test.go

```diff
@@ -194,6 +195,7 @@ func TestInfoHandler(t *testing.T) {
 	],
 	"client_drop_p0s": true,
 	"span_meta_structs": true,
+	"long_running_spans": true,
```

Comment:
We put the information here just to avoid adding coupling with the tracer (e.g maintaining a correspondence between agent version and feature it supports). Also, it looks like custom agent have no versions. May we keep this?